### PR TITLE
Win dynamic desktop: Add version 3.3.0

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/win-dynamic-desktop.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "version": "3.3.0",
+    "homepage": "https://github.com/t1m0thyj/WinDynamicDesktop",
+    "description": "Dynamic Desktop.",
```
Comment (Ash258) [inline bucket/win-dynamic-desktop.json]: I have no idea, what does this app do.

Comment (linsui) [reply bucket/win-dynamic-desktop.json]: How about `Cycle through multiple wallpapers based on the times of sunrise and sunset.` ?

File: bucket/win-dynamic-desktop.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "}"
+    ],
+    "persist": [
+        "themes",
+        "WinDynamicDesktop.pth",
+        "settings.conf"
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/t1m0thyj/WinDynamicDesktop/releases/download/v$version/WinDynamicDesktopPortable.exe#/WinDynamicDesktop.exe"
```
Comment (Ash258) [inline bucket/win-dynamic-desktop.json]: ??

Missing architecture

File: bucket/win-dynamic-desktop.json (hunk 1 above)
Comment (Ash258) [reply bucket/win-dynamic-desktop.json]: I do not know, why are you asking, when you still push it without answer to your question.

File: bucket/win-dynamic-desktop.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "3.3.0",
+    "homepage": "https://github.com/t1m0thyj/WinDynamicDesktop",
+    "description": "Cycle through multiple wallpapers based on the times of sunrise and sunset.",
+    "license": "MPL-2.0",
+    "url": "https://github.com/t1m0thyj/WinDynamicDesktop/releases/download/v3.3.0/WinDynamicDesktopPortable.exe#/WinDynamicDesktop.exe",
+    "hash": "3478c32c24caf1811d3a7039227c9497a7e35aaed445deca09849a540faf8b4c",
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\WinDynamicDesktop.pth\")) {",
+        "    New-Item \"$dir\\WinDynamicDesktop.pth\" -Value \"$(current_dir $dir)\" | Out-Null",
```
Comment (Ash258) [inline bucket/win-dynamic-desktop.json]: I do not think that string with subshell is needed.

```suggestion
        "    Set-Content \"$dir\\WinDynamicDesktop.pth\" (current_dir $dir) -Ecoding ASCII",
```

File: bucket/win-dynamic-desktop.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        "}",
+        "if (!(Test-Path \"$persist_dir\\settings.conf\")) {",
```
Comment (Ash258) [inline bucket/win-dynamic-desktop.json]: One line is 120 chars long. Github PR is 127 and github code view is 140.

```suggestion
        "if (!(Test-Path \"$persist_dir\\settings.conf\")) { New-Item \"$dir\\settings.conf\" -Value '{}' | Out-Null }"
```

**Pull request merged**